.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
+updates:
+  - package-ecosystem: "pip"
+    directory: "/docs"
+    schedule:
+      interval: "daily"
+    ignore:
+      - dependency-name: "*"
+    allow:
+      - dependency-name: "sphinx-scylladb-theme"
+      - dependency-name: "sphinx-multiversion-scylla"
```

Comment:
Iirc we didn't have that good of an experience with depandabot and had to manually fix the PRs either way, so I'm personally not a fan of adding this. Imo it shouldn't be mixed in this PR either way.

Reply:
+1